debugstatus/status_test.go

```diff
@@ -37,6 +38,13 @@ func (s *statusSuite) TestCheck(c *gc.C) {
 		makeCheckerFunc("check2", "check2 name", "value2", false),
 		makeCheckerFunc("check3", "check3 name", "value3", true),
 	)
+	for key, r := range results {
+		if r.Duration < time.Microsecond {
```

Comment:
If I were doing this test in Python I'd mock time.Now and time.Since to produce deterministic results. Putting a sleep in a test, even for a microsecond, has a certain code smell. Is there not a better way?